client/src/components/relatedItems/related.test.jsx

```diff
@@ -11,7 +12,16 @@ import ReactDOM from 'react-dom';
 //   expect(conElement).toBeInTheDocument();
 // });
 
-it('should render component', () => {
-  const {getByTestId} = render(<Related />);
-  expect(getByTestId('con-1')).toBeInTheDocument();
-})
+afterEach(cleanup);
+
+describe('rendering of components', () => {
+  it('should render component', () => {
+    const {getByTestId} = render(<Related />);
+    expect(getByTestId('con-1')).toBeInTheDocument();
+  });
+
+  // it('should render related items carousel', () => {
```

Comment:
Please look to omit these commented out lines prior to deployment. I understand our tests are still in development so not a necessity right now.

Reply:
noted, thanks!